src/Core/Engine/Renderer/API/OpenGL/FBO.cpp

```
@@ -138,6 +145,16 @@ void FBO::Bind()
     glBindFramebuffer(GL_FRAMEBUFFER, id);
 }
 
+float FBO::GetWidth()
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)